submissions/Harnytskyi/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,635 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Cinema Ticket Booking</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="css/style.css" />
+</head>
+
+<body>
+    <main>
+        <div class="booking_block">
+            <div id="booking_wrapper" class="booking_wrapper">
+                <div class="hall_container">
+                    <div class="hall">
+                        <div class="block_item"></div>
+                        <div class="block_item screen">SCREEN</div>
```

Comment:
If you need uppercase text here use CSS for it. Because when someone changed text, will it be uppercase?